### PR TITLE
fix(_run): wrong path to client/server cache directories

### LINKED ISSUE #1287: Fix and ensure successful operations with "Single" Dev environment deployment config
Deployment of a "[Single](https://github.com/ovrclk/akash/tree/v0.10.2/_run/single)" dev environment configuration currently fails (due to reasons set to be described in this issue).

This task entails identifying and resolving all deployment and general operational errors in order to aid in developer/community testing and other development processes.

### PR DESCRIPTION
**Pull request by troian**
fixes #1287

### CODE REVIEW

**Review by hydrogen18 — APPROVED** [review]

**Review by 0x0I — APPROVED** [review]

File: _run/common-kind.mk [hunk 1]
```diff
@@ -40,7 +42,7 @@ kind-k8s-ip:
 
 .PHONY: kind-configure-image
 kind-configure-image:
-	echo "- op: replace\n  path: /spec/template/spec/containers/0/image\n  value: ${DOCKER_IMAGE}" > ./kustomize/akash-node/docker-image.yaml && \
+	echo "- op: replace\n  path: /spec/template/spec/containers/0/image\n  value: $(DOCKER_IMAGE)" > ./kustomize/akash-node/docker-image.yaml && \
```
Comment (boz) [inline _run/common-kind.mk:45]: Can we just have the files there with this default so that this command doesn't need to be run?

**Review by boz — APPROVED** [review]
:rocket:

File: _run/common-kind.mk (hunk 1 above)
Comment (troian) [reply _run/common-kind.mk:45]: 💯

File: _run/single/kustomize/akash-node/docker-image.yaml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+- op: replace
+  path: /spec/template/spec/containers/0/image
+  value: ghcr.io/ovrclk/akash:latest
```
Comment (boz) [inline _run/single/kustomize/akash-node/docker-image.yaml:3]: `:stable`?

**Review by boz — APPROVED** [review]
:rocket:

**Pull request merged**